### ISSUE TITLE
Document that F5TTS_v1_Base/model_1250000.safetensors contains only EMA-prefixed weights

### ISSUE THREAD

**Issue by netlinux-ai** [SWivid/F5-TTS#1292]
The published checkpoint at `hf://SWivid/F5-TTS/F5TTS_v1_Base/model_1250000.safetensors` contains 366 tensors. Of those:

- 364 are under prefix `ema_model.transformer.*` — the EMA shadow weights
- 2 are bookkeeping (`step`, `initted`) for EMA state tracking
- **0 weights exist under any non-`ema_model.` prefix**

There is no non-EMA copy of the model weights in this file.

## Why this matters

Anyone writing a custom loader (rather than going through `f5_tts.api.F5TTS`'s built-in loading) gets surprising results:

```python
# naive: skip EMA prefix to get "real" weights
state = load_file("model_1250000.safetensors")
cleaned = {k: v for k, v in state.items() if not k.startswith("ema_model.")}
model.load_state_dict(cleaned, strict=False)
# missing: 364, unexpected: 2
# (model is now random-initialised, training/inference will fail silently)
```

The fix is to invert the filter: strip the prefix and keep the EMA weights:

```python
cleaned = {k[len("ema_model."):]: v
           for k, v in state.items()
           if k.startswith("ema_model.")}
model.load_state_dict(cleaned, strict=False)
# missing: 0, unexpected: 0
```

The published `f5_tts.api.F5TTS` class handles this internally via the `use_ema=True` flag, but the convention is not documented anywhere in the README or HuggingFace model card. While writing a minimal trainer to fine-tune on a custom corpus, we spent an evening assuming the model was random-initialised because step-1 loss came out at ~4.0 instead of the ~0.5 you'd expect from a working pretrained start. Stripping the prefix and using the EMA weights as the starting point produces the correct loss range.

## Suggested resolution

A note in any of:
- the model card on HuggingFace at `SWivid/F5-TTS`
- the training README in this repo
- a docstring on the `F5TTS` class explaining the structure

stating something like:

> The published checkpoint contains only EMA shadow weights (under prefix `ema_model.transformer.`). The non-EMA copy is not preserved in the public artefact. If you write a custom loader, strip the prefix and use these as the starting weights — they are the canonical inference and fine-tune starting point.

## Reproduction

```python
from safetensors.torch import load_file
from cached_path import cached_path
ckpt = str(cached_path("hf://SWivid/F5-TTS/F5TTS_v1_Base/model_1250000.safetensors"))
state = load_file(ckpt)
prefixes = {k.split(".", 1)[0] for k in state}
print(prefixes)  # {'ema_model', 'step', 'initted'}
```

## Versions tested

- F5-TTS 1.1.20 from PyPI
- safetensors 0.7.0

**Comment by Vidit-lab** [comment]
I'd like to work on documentation updates for this issue.
